docs/tutorial.rst

```diff
@@ -168,6 +230,9 @@ then download the coverage track.
    # Download the coverage track
    bigwig.download()
 
+Writing a new process
```

Comment:
`- slug: max-reads            # User readable unique identifier
  name: Max reads            # Process name
  requirements:
    expression-engine: jinja
  data_name: Chromosome      # The name of data objects created by the process
  version: 1.0.0             # Process version
  type: data:bam:stats       # The type of the process and data objects created by it
  input:
    - name: bam
      label: BAM file
      type: data:alignment:bam
  output:
    - name: stats
      label: Stats
      type: basic:file
    - name: chr
      label: Chr (max reads)
      type: basic:string
  run:                       # The algorithm in bash using Django template tags
    runtime: polyglot
    language: bash
    program: |
      # Compute sequence alignment statistics
      samtools idxstats '{{bam.bam.file}}' > stats.txt

      # Save the statistics file
      re-save-file stats stats.txt

      # Find chromosome with the largest number of aligned reads
      CHR=`<my_script>.py stats.txt`

      # Save the chromosome name
      re-save chr "${CHR}"`